save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -0,0 +1,201 @@
+ * 1) to put new roles of users
+ * 2) to get users and their roles by organization
+ * 3) to remove users from organizations
+ */
+
+package org.cqfn.save.backend.controllers
+
+import org.cqfn.save.backend.security.OrganizationPermissionEvaluator
+import org.cqfn.save.backend.service.LnkUserOrganizationService
+import org.cqfn.save.backend.service.OrganizationService
+import org.cqfn.save.backend.utils.AuthenticationDetails
+import org.cqfn.save.domain.Role
+import org.cqfn.save.info.UserInfo
+import org.cqfn.save.permission.Permission
+import org.cqfn.save.permission.SetRoleRequest
+import org.cqfn.save.v1
+
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.server.ResponseStatusException
+
+/**
+ * Controller for processing links between users and their roles in organizations
+ */
+@RestController
+@RequestMapping("/api/$v1/organizations/")
+class LnkUserOrganizationController(
+    private val lnkUserOrganizationService: LnkUserOrganizationService,
+    private val organizationService: OrganizationService,
+    private val organizationPermissionEvaluator: OrganizationPermissionEvaluator,
+) {
+    private val logger = LoggerFactory.getLogger(LnkUserOrganizationController::class.java)
+
+    /**
+     * @param organizationName
+     * @param authentication
+     * @return list of users with their roles, connected to the organization with [organizationName]
+     * @throws ResponseStatusException
+     */
+    @GetMapping("{organizationName}/users")
+    fun getAllUsersByOrganizationName(
+        @PathVariable organizationName: String,
+        authentication: Authentication,
+    ): List<UserInfo> {
+        val organization = organizationService.findByName(organizationName)
+            ?: throw ResponseStatusException(HttpStatus.NOT_FOUND)
+        val usersWithRoles = lnkUserOrganizationService.getAllUsersAndRolesByOrganization(organization)
+            .filter { (_, role) -> role != Role.NONE }
+            .map { (user, role) ->
+                user.toUserInfo(organizations = mapOf(organization.name to role))
+            }
+            .also { logger.trace("Found ${it.size} users for organization: $it") }
+        return usersWithRoles
+    }
+
+    /**
+     * @param organizationName
+     * @param userName
+     * @param authentication
+     * @return role of user with [userName] in organization with [organizationName]
+     * @throws ResponseStatusException
+     */
+    @GetMapping("/{organizationName}/users/roles")
+    @ApiResponse(responseCode = "200", description = "Successfully fetched user's role")
+    @ApiResponse(
+        responseCode = "404", description = "Requested user or organization doesn't exist."
+    )
+    fun getRole(
```

Comment:
This is duplicate fun _getRoleOrganization_ in OrganizationController

Reply:
Ok, removed `getRoleOrganization` from `OrganizationController`